homeassistant/components/hunterdouglas_powerview/sensor.py

```diff
@@ -31,19 +41,41 @@ async def async_setup_entry(hass, entry, async_add_entities):
     coordinator = pv_data[COORDINATOR]
     device_info = pv_data[DEVICE_INFO]
 
+    battery_sensors = entry.options.get(
```

Comment:
We usually create the entities as disabled by default instead of providing config options 

https://developers.home-assistant.io/docs/entity_registry_disabled_by/